server/controllers/profile-controller.js

```diff
@@ -0,0 +1,71 @@
+import { successResponse, errorResponse } from '../utils/helpers';
+import db from '../models';
+
+const { User } = db;
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
+ */
+class ProfileController {
+  /**
+   * get a particular author
+   * @static
+   * @param {Request} req request object
+   * @param {Response} res response object
+   * @memberof {Users}
+   * @returns {undefined} .
+   */
+  static async getProfile(req, res) {
+    const { username } = req.params;
+    try {
+      return User.findOne({ where: { username } })
+        .then(data => {
+          if (!data) {
+            errorResponse(res, 'username does not exist');
+          }
+          successResponse(
+            res,
+            {
+              user: data
+            },
+            200
+          );
+        })
+        .catch(error => res.send(error.message));
+    } catch (error) {
+      errorResponse(res, error.message);
+    }
+  }
+
+  /**
+   * get all author
+   * @static
+   * @param {Request} req request object
+   * @param {Response} res response object
+   * @memberof {Users}
+   * @returns {undefined} .
+   */
+  static async getAll(req, res) {
+    try {
+      return User.findAll()
+        .then(data => {
+          if (!data) {
+            errorResponse(res, 'No User(s)', 400);
+          }
+          successResponse(
+            res,
+            {
+              user: data
```

Comment:
Again no filtering of the user record - please correct

Reply:
please I don't understand what you mean by filtering user record